src/core/featuremodel.cpp

```diff
@@ -747,6 +784,24 @@ bool FeatureModel::create()
       QgsFeature feat;
       if ( mLayer->getFeatures( QgsFeatureRequest().setFilterFid( createdFeatureId ) ).nextFeature( feat ) )
       {
+        // Revisit relations in need of attribute updates
+        for ( auto revisitRelation : revisitRelations )
```

Comment:
```suggestion
        for ( const QPair<QgsRelation, QgsFeatureRequest> revisitRelation : std::as_const( revisitRelations ) )
```

I find this more reviwable and readable in the long term, no strong opinion.

Reply:
In any case, it should be const?